deploy/aws/README.md

```diff
@@ -22,7 +22,7 @@ export KEYNAME=YOUR_KEY # Name of an existing EC2 KeyPair to enable SSH access t
 export SUBNET=YOUR_AWS_SUBNET # Subnet that EC2 instances will be created in. Can be found from AWS console or running command: aws ec2 describe-subnets
```

Comment:
Could you update the aws readme and remove the subnet references (the 'aws cloudformation create-stack' command fails since it references SubnetID which doesn't exist in the template anymore)